Dockerfile

```diff
@@ -205,6 +207,7 @@ RUN apk --update --no-cache add \
     brotli \
     ca-certificates \
     coreutils \
+    cppunit-dev \
```

Comment:
I don't think this is needed but if it is it should move to a deps dependency and removed later.

Reply:
This binary was required to make rTorrent run. We need to patch rTorrent to eliminate it.

Reply:
Ok let's keep it for now, can look at this in follow-up